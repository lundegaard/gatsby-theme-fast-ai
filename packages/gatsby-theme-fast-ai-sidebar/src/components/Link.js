import React from 'react';
import { Link as IntlLink } from 'gatsby-plugin-intl';
import { Link as GatsbyLink } from '@reach/router';
import { Link as ComponentsLink } from '@fast-ai/ui-components';

const Link = (props) => <ComponentsLink as={IntlLink} {...props} />;

export const NoIntlLink = (props) => <ComponentsLink as={GatsbyLink} {...props} />;

export default Link;
