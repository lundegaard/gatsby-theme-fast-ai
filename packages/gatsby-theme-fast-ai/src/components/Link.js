import React from 'react';
import { Link as GatsbyLink } from 'gatsby-plugin-intl';
import { Link as ComponentsLink } from '@fast-ai/ui-components';

const Link = props => <ComponentsLink as={GatsbyLink} {...props} />;

export default Link;
