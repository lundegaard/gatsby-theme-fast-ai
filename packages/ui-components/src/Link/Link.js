import React from 'react';
import { Link as RebassLink } from 'rebass';

const Link = props => <RebassLink {...props} />;

export default Link;
