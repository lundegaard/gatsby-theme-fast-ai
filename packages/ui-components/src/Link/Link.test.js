import React from 'react';
import { shallow } from 'enzyme';

import Link from './Link';

describe('<Link />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<Link {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
