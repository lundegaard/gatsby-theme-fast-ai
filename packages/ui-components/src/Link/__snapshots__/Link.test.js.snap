// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> matches snapshot when shallowly rendered 1`] = `<ForwardRef />`;
